internal/controller/constants.go

```diff
@@ -98,4 +98,9 @@ const (
 
 	/*** watchers ***/
 	WatcherAnnotationKey = "ols.openshift.io/watcher"
+	// TelemetryPullSecretNamespace "openshift-config" contains the telemetry pull secret to determine the enablement of telemetry
+	// #nosec G101
+	TelemetryPullSecretNamespace = "openshift-config"
+	// TelemetryPullSecretName is the name of the secret containing the telemetry pull secret
+	TelemetryPullSecretName = "pull-secret"
```

Comment:
don't you need a new role+rolebinding to grant the operator permission to read/watch this resource?

Reply:
the role is here https://github.com/openshift/lightspeed-operator/pull/153/files#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7R82-R93

Reply:
during the installation from bundle, a clusterrole is created with more than the required access rights on secret.
It seems more rights  granted than we wanted though. 
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: "2024-06-11T08:35:46Z"
  labels:
    olm.managed: "true"
    olm.owner: lightspeed-operator.v0.0.1
    olm.owner.kind: ClusterServiceVersion
    olm.owner.namespace: openshift-lightspeed
    olm.permissions.hash: 4ocN4MAqabrmojboiME86jksllfmx47XTsRpBP
    operators.coreos.com/lightspeed-operator.openshift-lightspeed: ""
  name: lightspeed-operator.v0.0-5Aqfwi9a4ZKSKdIsDy4AsLMj5iQ6GUFBB15dIZ
  resourceVersion: "42573"
  uid: 021a457a-e621-4fbb-8527-d0a5f94059b6
rules:
- apiGroups:
  - ""
  resources:
  - configmaps
  verbs:
  - get
  - list
  - watch
  - create
  - update
  - patch
  - delete
- apiGroups:
  - coordination.k8s.io
  resources:
  - leases
  verbs:
  - get
  - list
  - watch
  - create
  - update
  - patch
  - delete
- apiGroups:
  - ""
  resources:
  - events
  verbs:
  - create
  - patch
- apiGroups:
  - apps
  resources:
  - deployments
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - ""
  resources:
  - configmaps
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - ""
  resources:
  - secrets
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - ""
  resources:
  - serviceaccounts
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - ""
  resources:
  - services
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - monitoring.coreos.com
  resources:
  - prometheusrules
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - monitoring.coreos.com
  resources:
  - servicemonitors
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - rbac.authorization.k8s.io
  resources:
  - rolebindings
  - roles
  verbs:
  - '*'
- apiGroups:
  - ""
  resources:
  - namespaces
  verbs:
  - get
  - list
  - watch
```

Reply:
> the role is here https://github.com/openshift/lightspeed-operator/pull/153/files#diff-6d4c4e29ec096149781536bb9118fde8242c1901df764e3eda3019ac8503d6c7R82-R93

what about the rolebinding that uses the role?